x-pack/plugins/lists/common/schemas/request/create_endpoint_list_item_schema.test.ts

```diff
@@ -65,7 +65,7 @@ describe('create_endpoint_list_item_schema', () => {
     expect(message.schema).toEqual({});
   });
 
-  test('it should not validate a "list_id" since it does not required one', () => {
+  test('it should fail validation when supplied a "list_id" since it does not required one', () => {
```

Comment:
@rylnd you'd mentioned the wording `not validate` was a bit misleading, so began updating the wording. Does this seem better?